src/Compilers/CSharp/Portable/Binder/Semantics/BestTypeInferrer.cs

```diff
@@ -222,14 +211,33 @@ private static TypeSymbol GetBestType(ArrayBuilder<TypeSymbol> types, Conversion
                 }
             }
 
+            if (includeNullability)
```

Comment:
>includeNullability [](start = 16, length = 18)

As we discussed, adding a comment (why is this merging separate from others, consider moving others here for regularity) would be nice. Thanks